data/announcements/scheduled.yaml

```
@@ -232,3 +232,18 @@ announcements:
     <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconeu24&utm_content=slim-banner">
     Buy your ticket now! 19 - 22 March | Paris, France
     </a> 
+
+- name: KubeCon 2024 China
+  startTime: 2024-07-26T00:00:00  #Added in https://github.com/kubernetes/website/pull/44828
```

The PR number in the comment will need to be updated to reflect the correct number for this pull request.

```suggestion
  startTime: 2024-07-26T00:00:00  #Added in https://github.com/kubernetes/website/pull/46431
```